cmd/helm/main.go

```diff
@@ -127,31 +130,25 @@ func main() {
 	}
 
 	ns := util.PodNamespace(systemNamespace)
-	storageURL, err := url.Parse(fmt.Sprintf("%s/bundles/", httpExternalAddr))
-	if err != nil {
-		setupLog.Error(err, "unable to parse bundle content server URL")
-		os.Exit(1)
-	}
-
-	localStorage := &storage.LocalDirectory{
-		RootDirectory: storageDirectory,
-		URL:           *storageURL,
-	}
-
+	var bundleCA io.Reader
 	var rootCAs *x509.CertPool
 	if bundleCAFile != "" {
+		bundleCA, err = os.Open(bundleCAFile)
+		if err != nil {
+			setupLog.Error(err, "unable to create storage")
+			os.Exit(1)
+		}
 		var err error
-		if rootCAs, err = util.LoadCertPool(bundleCAFile); err != nil {
+		if rootCAs, err = util.LoadCertPool(bundleCA); err != nil {
 			setupLog.Error(err, "unable to parse bundle certificate authority file")
 			os.Exit(1)
 		}
```

Comment:
This logic might be a potential candidate for a dedicated function if we're using it throughout multiple provisioner implementations.

Reply:
I created a function in `internal/util/util.go` for this logic.